api/src/db/migrations/v131_add_construction_type_essential_question.ts

```diff
@@ -0,0 +1,480 @@
+import { v130Business, v130UserData } from "@db/migrations/v130_remove_nexus_location_in_new_jersey";
+
+export interface v131IndustrySpecificData {
+  readonly liquorLicense: boolean;
+  readonly requiresCpa: boolean;
+  readonly homeBasedBusiness: boolean | undefined;
+  readonly providesStaffingService: boolean;
+  readonly certifiedInteriorDesigner: boolean;
+  readonly realEstateAppraisalManagement: boolean;
+  readonly cannabisLicenseType: v131CannabisLicenseType;
+  readonly cannabisMicrobusiness: boolean | undefined;
+  readonly constructionRenovationPlan: boolean | undefined;
+  readonly carService: v131CarServiceType | undefined;
+  readonly interstateLogistics: boolean;
+  readonly interstateMoving: boolean;
+  readonly isChildcareForSixOrMore: boolean | undefined;
+  readonly petCareHousing: boolean | undefined;
+  readonly willSellPetCareItems: boolean | undefined;
+  readonly constructionType: v131ConstructionType;
+  readonly residentialConstructionType: v131ResidentialConstructionType;
+}
+
+export const migrate_v130_to_v131 = (v130Data: v130UserData): v131UserData => {
+  return {
+    ...v130Data,
+    businesses: Object.fromEntries(
+      Object.values(v130Data.businesses)
+        .map((business: v130Business) => migrate_v130Business_to_v131Business(business))
+        .map((currBusiness: v131Business) => [currBusiness.id, currBusiness])
+    ),
+    version: 131,
+  } as v131UserData;
+};
+
+const migrate_v130Business_to_v131Business = (business: v130Business): v131Business => {
+  return {
+    ...business,
+    profileData: {
+      ...business.profileData,
+      constructionType: undefined,
+      residentialConstructionType: undefined,
+      interstateLogistics: business.profileData.isInterstateLogisticsApplicable,
```

Comment:
This is to have the `ProfileData` described in migrations and the `ProfileData` described in `ProfileData.tsx` back in alignment.

Reply:
good call out

Reply:
Update: Ended up keeping the status quo for now  because I realized the fix might not fully combine the two sources of truth for this data. Added a ticket to resolve separately